tests/test_cli.py

```diff
@@ -1016,11 +1037,14 @@ def verify_noQC_citations_not_written(self, cmd, caplog):
         # shouldn't be offered to the user.
         out, err = capture_output(stdpopsim.cli.stdpopsim_main, cmd.split())
         log_output = caplog.text
+        # citations are written out via the logging mechanism
+        assert len(out) == 0
+        assert len(err) == 0
         for citation in self.model.citations:
-            assert not (citation.author in out)
-            assert not (citation.doi in out)
-            assert not (citation.author in err)
-            assert not (citation.doi in err)
+            assert not (citation.author in log_output)
```

Comment:
In figuring out how this works, I discovered we weren't doing what we wanted in this test.